BuildAutoRest.ps1

```diff
@@ -125,8 +126,11 @@ ForEach ($SDK In $SDKName)
         {
             If (Test-Path -Path:($OutputFullPath)) { Get-ChildItem -Path:($OutputFullPath) | Where-Object { $_.Name -notin $FolderExcludeList } | Remove-Item -Force -Recurse }
             If (!(Test-Path -Path:($OutputFullPath))) { New-Item -Path:($OutputFullPath) -ItemType:('Directory') }
+            Write-Host ('[RUN COMMAND] Clean Script') -BackgroundColor:('Black') -ForegroundColor:('Magenta')
+            bash $CleanScript $PSScriptRoot $OutputFullPath
             Write-Host ('[RUN COMMAND] autorest ' + $ConfigFileFullName + ' --force --verbose --debug') -BackgroundColor:('Black') -ForegroundColor:('Magenta')
-            autorest $ConfigFileFullName --force --verbose --debug | Tee-Object -FilePath:($LogFilePath) -Append
+            autorest $ConfigFileFullName --force --version="3.6.6" --use:@autorest/powershell@3.0.463 | Tee-Object -FilePath:($LogFilePath) -Append
```

Comment:
I'm pinning the version of Autorest, Autorest Core/ Powershell to prevent issues from cropping up later

Reply:
I dont think you need to pin it here as your already doing it in SetupDependencies.